packages/ui/hooks/index.ts

```diff
@@ -158,10 +158,10 @@ export function useExpiringLocalPreference<Data = any>(
             if (validData !== null && expiresAt && Date.now() < expiresAt) {
                setData(validData);
             } else {
-               localStorage.deleteIem(key);
```

Comment:
Dang I can't believe we were using a nonexistent `deleteIem` method and silencing the error.